TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/full_tests.py

```diff
@@ -113,14 +131,15 @@ def test_e2e_0001_default(self):
                         expected_status_code=HTTPStatus.CREATED, test_case=self)
 
         ignore_list = [".", "searchguard", "sg7", "security-auditlog", "reindexed-logs"]
-        expected_docs = {}
+        expected_source_docs = {}
+        expected_target_docs = {}
         # Source should have both documents
-        expected_docs[index_name] = {"count": 2}
-        check_doc_counts_match(cluster=source_cluster, expected_index_details=expected_docs,
+        expected_source_docs[index_name] = {"count": 2}
+        check_doc_counts_match(cluster=source_cluster, expected_index_details=expected_source_docs,
                                index_prefix_ignore_list=ignore_list, test_case=self)
         # Target should have one document from snapshot
-        expected_docs[index_name] = {"count": 1}
-        check_doc_counts_match(cluster=target_cluster, expected_index_details=expected_docs,
+        expected_target_docs[transformed_index_name] = {"count": 1}
+        check_doc_counts_match(cluster=target_cluster, expected_index_details=expected_target_docs,
```

Comment:
I must be missing something here, as I'd expect this check to fail.  It appears the index name in the snapshot is the original, and we're passing in a transformation to the backfill process.  As a result, I'd expect that the backfill process would create a new index on the target with default settings and the original index name, and put its doc into that.

Can you fill me in on how this check is passing?